.github/workflows/build-release.yml

```diff
@@ -35,14 +35,27 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install wheel setuptools
+        pip install wheel setuptools rjsmin rcssmin
     
     - name: Build tar.gz
       run: python3 setup.py sdist
     
     - name: Build wheel
-      run: python3 setup.py bdist_wheel
-    
+      run: |
+        bundlejs="$(sha224sum assets/scripts/*.js | sha224sum | cut -d ' ' -f1).js"
+        bundlecss="$(sha224sum assets/styles/*.css | sha224sum | cut -d ' ' -f1).css"
```

Comment:
I generate an unique URL using a sha2sum, I could have used a non-cryptographic hash function but `sha224sum` is fast to compute and is available out-of-box (it comes with `coreutils`). I use sha2 because if I had use md5/sha1 people would have (incorrectly, has we don't need a *cryptographic* hash function) reported a security issue "gnagna it is unsafe gnagna). I use the 224 variants because I don't need much bytes to ensure there is no collision and because I like short URLs

Reply:
Human signification : sha2sum all assets (it gives one sha2sum per file), sha2sum the output (to have only one hash) and take the hash (the output give the hash and the filename separated by a blank)